src/NUnitFramework/framework/Attributes/OneTimeSetUpAttribute.cs

```diff
@@ -26,8 +26,8 @@ namespace NUnit.Framework
 	using System;
 
 	/// <summary>
-	/// Attribute used to identify a method that is called once
-	/// to perform setup before any child tests are run.
+	/// Identifies a method to be called once, prior to the child tests executing, 
+	/// to perform any setup requirements for the proceding tests.
```

Comment:
*preceding - but I think this is the wrong word anyway. Maybe you mean 'descendant' tests - but is that perhaps overcomplicating the explanation? Can you clarify what you wanted to change about the previous description here, and maybe we can come up with a better summary?